drivers/eeprom/eeprom_at2x.c

```diff
@@ -201,29 +209,58 @@ static size_t eeprom_at2x_size(const struct device *dev)
 }
 
 #ifdef CONFIG_EEPROM_AT24
+
+static uint16_t eeprom_at24_translate_offset(const struct device *dev,
+					     off_t *offset)
+{
+	const struct eeprom_at2x_config *config = dev->config;
+
+	const uint16_t addr_incr = *offset >> config->addr_width;
```

Comment:
Looks good, thank you. Could you please add a comment (in the source code) explaining that these additions are for handling AT24Cxx as continuous memory?

Reply:
I added a comment block before the function